CHANGELOG.md

```diff
@@ -1,8 +1,68 @@
 See full changelog for the OpenAPI schema (OAS) [here](https://github.com/plaid/plaid-openapi/blob/master/CHANGELOG.md).
 
-# 11.9.1
```

Comment:
did you mean to delete this from the changelog?

Reply:
I did not, good catch!